packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -0,0 +1,321 @@
+                let rows = files.map((file, index) => ({name: file.name, type: file.type, id: index}));
+                setFilledSrc(rows);
+                setIsFilled(true);
+              }}>
+              <Button variant="primary">Select a file</Button>
+            </FileTrigger>
+          </Content>
+        </IllustratedMessage>
+      </DropZone>
+      <TableView
+        renderEmptyState={renderEmptyState}
+        height="size-3000"
+        minWidth="size-3000">
+        <TableHeader columns={columns}>
+          {column => (
+            <Column
+              key={column.uid}>
+              {column.name}
+            </Column>)}
+        </TableHeader>
+        {isFilled ? 
+          <TableBody items={filledSrc}>
+            {item => (
+              <Row>
+                {columnKey => <Cell>{item[columnKey]}</Cell>}
+              </Row>
+            )}
+          </TableBody> :
+          <TableBody>
+            {[]}
+          </TableBody>
+        }
+      </TableView>
+    </>
+  );
+}
+
+function DropZoneWithDraggable(props) {
+  let [isFilled, setIsFilled] = useState(false);
+  let [filledSrc, setFilledSrc] = useState(null);
+
+  return (
+    <>
+      <Draggable />
+      <DropZone 
+        {...props}
+        isFilled={isFilled}
+        onDrop={async (e) => {
+          let items = await Promise.all(e.items.filter((item) => item.kind === 'text' && item.types.has('text/plain')).map((item: TextDropItem) => item.getText('text/plain')));
+          if (items.length > 0) {
+            setIsFilled(true);
+            setFilledSrc(items.join('\n'));
+          }
+        }}
+        onDropEnter={action('onDropEnter')}
+        onDropExit={action('onDropExit')} 
+        onPaste={action('onPaste')}>
+        <IllustratedMessage>
+          <Upload />
+          <Heading>
+            <Text slot="label">
+              Drag and Drop here
+            </Text>
+          </Heading>
+        </IllustratedMessage>
+        {filledSrc}
+      </DropZone>
+    </>
+  );
+}
+
+function DropZoneWithButton(props) {
+  let [isFilled, setIsFilled] = useState(false);
+  let [filledSrc, setFilledSrc] = useState(null);
+
+  return (
+    <>
+      <DropZone 
+        {...props}
+        width="size-3000"
+        height="size-3000"
+        isFilled={isFilled}
+        UNSAFE_className={classNames(styles, isFilled && 'is-filled')}
+        onDrop={async (e) => { 
+          let item = e.items.find((item) => item.kind === 'file') as FileDropItem;
+          if (item) {
+            setFilledSrc({
+              src: URL.createObjectURL(await item.getFile()), 
+              type: item.type,
+              name: item.name});
+            setIsFilled(true);
+          }
+        }}
+        onDropEnter={action('onDropEnter')}
+        onDropExit={action('onDropExit')} 
+        onPaste={action('onPaste')}>
+        <IllustratedMessage>
+          <Upload />
+          <Heading>
+            <Text slot="label">
```

Comment:
The stories have been updated so that for each there is some textual content used to label the drop zone (except for the `is-filled` story). Hopefully that will reduce confusion regarding if the story is making the correct announcements.